discovery-infra/test_infra/consts/consts.py

```diff
@@ -28,6 +28,7 @@ class NetworkType:
 WORKER_TF_CPU_MODE = HOST_PASSTHROUGH_CPU_MODE
 NODES_REGISTERED_TIMEOUT = 20 * MINUTE
 DEFAULT_CHECK_STATUSES_INTERVAL = 5
+CLUSTER_READY_FOR_INSTALL_TIMEOUT = 10 * MINUTE
```

Comment:
Is this a reasonable value? This is effectively reducing the time from an hour to 10 minutes, right?

Reply:
The other timeout was waiting for complete install, now it's just waiting for the agents to be ready for install, so 10m should be enough